src/material/list/list-option.ts

```diff
@@ -105,8 +112,21 @@ export class MatListOption extends MatListItemBase implements ListOption, OnInit
   @Output()
   readonly selectedChange: EventEmitter<boolean> = new EventEmitter<boolean>();
 
-  /** Whether the label should appear before or after the checkbox. Defaults to 'after' */
-  @Input() checkboxPosition: MatListOptionCheckboxPosition = 'after';
+  /** Whether the label should appear before or after the checkbox/radio. Defaults to 'after' */
+  @Input() togglePosition: MatListOptionTogglePosition = 'after';
+
+  /**
+   * Whether the label should appear before or after the checkbox/radio. Defaults to 'after'
+   *
+   * @deprecated Use `togglePosition` instead.
```

Comment:
This PR is labeled `target: rc` - if it actually does make it into RC you don't have to bother deprecating things, you can just make the change (since this is an unreleased component and has no users). However, if it slips so that its not shipping with v15.0.0 we would need to go through the usual deprecation stuff